src/Analyzer.Utilities/Analyzer.Utilities.csproj

```diff
@@ -26,6 +26,9 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
   </PropertyGroup>
+  <PropertyGroup>
```

Comment:
Note that this (and other such changes) is from a separate commit that will never be merged.